.travis.yml

```diff
@@ -275,6 +269,43 @@ matrix:
         # https://docs.travis-ci.com/user/deployment#uploading-files-and-skip_cleanup.
         skip_cleanup: true
       <<: *extended-test-suite
+    # See detailed reasoning for snap tests configuration just above.
+    - env: SNAP=true TARGET_ARCH=arm64
+      dist: bionic
```

Comment:
Yes we could use a YAML anchor to inject the config for each relevant item. However, it is at the cost of indirections in how you read the `travis.yml` file as a human, since this anchor will be at the beginning of the YAML file. If I think it is worth it for the job conditions, that are used a dozen of time, it is less the case for 3 entries.

Reply:
Yeah I think it's a good idea, especially with 4 entries like you said. I'd personally rather have the indirection than the duplication.

Reply:
Done.